.github/contributing.md

```diff
@@ -74,29 +77,28 @@ time to [learn](https://help.github.com/categories/bootcamp/).
 
 ### When Programming
 We recommend using an IDE; you can find some set up instructions in README.
-Also, please follow our CODING_CONVENTIONS.
+Please also follow our code conventions.
 
 ### After Programming
 Test your changes. Actually, test more than your changes: if you think that you
 might have broken something unrelated, better to test that too. Nothing is more
 annoying than breaking existing features.
 
+After you have tested the changes, write automated test scripts if possible.
+They make it less likely for your changes to break in future releases, when
+other people work on Skript.
```

Comment:
Maybe documents regarding test scripts are needed or a pointer to example test scripts should be here

Reply:
I'll put a link to test script README and see if it needs improvements.